src/Dropdown.php

```diff
@@ -2675,6 +2675,12 @@ public static function getDropdownValue($post, $json = true)
                     $entities[] = (int) $value;
                 }
             }
+
+            if (empty($entities)) {
+                // Fall back to user's active entities if the entity array is empty after decoding
+                $entities = $_SESSION['glpiactiveentities'];
+            }
```

Comment:
If entities are empty, then there is probably something wrong on the caller side. It has to be fixed there. Forcing the current entity scope will just hide bugs, so it is a bad idea.